### ISSUE TITLE
Optimizer

### ISSUE THREAD

**Issue by pacorikos** [maziarraissi/PINNs#24]
This is a two-fold question.
Why are two optimizers used? One is used every iteration and the other is used after the 5,000 iterations. Secondly, does the AdamOptimizer need to be defined in the __init__ method, because I was interested in implementing a learning schedule and initializing a new RMSProp optimizer every iteration but I was wondering how I would do that with their setup.

**Comment by nish-ant** [comment]
For your first question, you can refer to my [response](https://github.com/maziarraissi/PINNs/issues/6#issuecomment-575896092) to another similar query.

You can try to implement learning schedule with the optimizer following the example [here](https://keras.io/api/optimizers/#learning-rate-decay--scheduling). I don't think you need to initialize the optimizer for each iteration when you use the scheduler that is provided with the module.
